apis/builder/validators.yaml

```
@@ -22,6 +22,9 @@ post:
           type: array
           items:
               $ref: "../../builder-oapi.yaml#/components/schemas/SignedValidatorRegistration"
+      application/octet-stream:
+        schema:
+          description: "SSZ serialized `List[SignedValidatorRegistrationV1]` bytes. Use content type header to indicate that SSZ data is contained in the request body."
```

```suggestion
          description: "SSZ serialized `List[SignedValidatorRegistrationV1, VALIDATOR_REGISTRY_LIMIT]` bytes. Use content type header to indicate that SSZ data is contained in the request body."
```

The SSZ list must have a defined maximum length. I suggest using [`VALIDATOR_REGISTRY_LIMIT`](https://github.com/ethereum/consensus-specs/blob/dev/specs/phase0/beacon-chain.md#state-list-lengths) (essentially unbounded) from the consensus-specs.

---

Good suggestion, I wasn't sure what limit to use so I left it up to implementation but you're right, we should have this in the spec.